src/query/storage/m3/dynamic_cluster_test.go

```diff
@@ -403,14 +393,19 @@ func TestDynamicClustersInitFailures(t *testing.T) {
 		},
 	}
 
-	opts := NewDynamicClusterOptions().
-		SetDynamicClusterNamespaceConfiguration([]DynamicClusterNamespaceConfiguration{cfg}).
-		SetInstrumentOptions(instrument.NewOptions())
+	opts := newTestOptions([]DynamicClusterNamespaceConfiguration{cfg})
 
 	_, err := NewDynamicClusters(opts)
 	require.Error(t, err)
 }
 
+func newTestOptions(cfgs []DynamicClusterNamespaceConfiguration) DynamicClusterOptions {
```

Comment:
nit; if you changed this to `newTestOptions(cfgs ...DynamicClusterNamespaceConfiguration)` , usage would go from `opts := newTestOptions([]DynamicClusterNamespaceConfiguration{cfg, cfg2})` to `opts := newTestOptions(cfg, cfg2)` :+1: